c++/src/kj/compat/tls-test.c++

```diff
@@ -1149,6 +1151,44 @@ KJ_TEST("TLS receiver does not stall on hung client") {
   KJ_EXPECT(!extraAcceptPromise.poll(test.io.waitScope));
 }
 
+#if !_WIN32 // TODO: Investigate and fix issue on Windows.
```

Comment:
Can you elaborate? We are going to be supporting Windows soon so ignoring it here might be an issue.